syft/messaging/plan/plan.py

```diff
@@ -439,6 +503,9 @@ def simplify(plan: "Plan") -> tuple:
             sy.serde._simplify(plan.state),
             sy.serde._simplify(plan.include_state),
             sy.serde._simplify(plan.is_built),
+            sy.serde._simplify(plan.input_shapes),
+            sy.serde._simplify(plan._output_shape) if plan._output_shape is not None else None,
```

Comment:
you can simplify None so sy.serde._simplify(plan._output_shape) should be sufficient